### PR TITLE
2.13.0 Release

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Prepare 2.13.0 release

Milestone: 2.13.0

Changelog: n/a

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -11641,10 +11641,10 @@ vite-plugin-checker@^0.6.2:
...
     vscode-uri "^3.0.2"
 
-vite@^4.4.9:
-  version "4.4.12"
-  resolved "https://registry.npmjs.org/vite/-/vite-4.4.12.tgz#e9c355d5a0d8a47afa46cb4bad10820da333da5c"
-  integrity sha512-KtPlUbWfxzGVul8Nut8Gw2Qe8sBzWY+8QVc5SL8iRFnpnrcoCaNlzO40c1R6hPmcdTwIPEDkq0Y9+27a5tVbdQ==
+vite@^4.4.12:
+  version "4.5.1"
+  resolved "https://registry.npmjs.org/vite/-/vite-4.5.1.tgz#3370986e1ed5dbabbf35a6c2e1fb1e18555b968a"
+  integrity sha512-AXXFaAJ8yebyqzoNB9fu2pHoo/nWX+xZlaRwoeYUxEqBO+Zj4msE5G+BhGBll9lYEKv9Hfks52PAF2X7qDYXQA==
```
Comment (traeok) [inline yarn.lock:11647]: I found it interesting that the lockfile is pointing to version `vite@4.5.1` even though the Dependabot PRs resolved Vite to version `vite@4.4.12`.

Either way, I don't see any breaking changes with Vite v4.5 so I think we can disregard this 😅

**Review by traeok — APPROVED** [review]
Thanks @JillieBeanSim for preparing the v2.13 release 🙂

File: yarn.lock (hunk 1 above)
Comment (JillieBeanSim) [reply yarn.lock:11647]: I noticed the change when running fresh-clone and yarn on main too, it is a dev dependency and has a caret.  I didn't see any issues testing either but we may want to watch adding the carets in the package.json dependencies.

**Review by zFernand0 — APPROVED** [review]
I have not found any showstoppers on the release.
However, there is one small item that I haven't had a chance to thoroughly test.
I'm referring to the addition of what seems to me like a new required method to the zowe-explorer-api package in the PR below:
- https://github.com/zowe/vscode-extension-for-zowe/pull/2602

That being said, I'm ok with continuing forward with the release.

LGTM! 😋

**Review by rudyflores — APPROVED** [review]
LGTM, thanks for making this PR Billie!

**Comment by JillieBeanSim** [comment]
> I have not found any showstoppers on the release. However, there is one small item that I haven't had a chance to thoroughly test. I'm referring to the addition of what seems to me like a new required method to the zowe-explorer-api package in the PR below:
> 
> * [add ability to add profile to all trees #2602](https://github.com/zowe/vscode-extension-for-zowe/pull/2602)
> 
> That being said, I'm ok with continuing forward with the release.
> 
> LGTM! 😋

@zFernand0 I can push a `?` for peace of mind before we go out 😄

**Comment by rudyflores** [comment]
> > I have not found any showstoppers on the release. However, there is one small item that I haven't had a chance to thoroughly test. I'm referring to the addition of what seems to me like a new required method to the zowe-explorer-api package in the PR below:
> > 
> > * [add ability to add profile to all trees #2602](https://github.com/zowe/vscode-extension-for-zowe/pull/2602)
> > 
> > That being said, I'm ok with continuing forward with the release.
> > LGTM! 😋
> 
> I can push a `?` for peace of mind before we go out 😄

I tested with ftp extension and other extender and wasn't seeing issues when creating this PR, but I agree maybe extra piece of mind lol

**Pull request merged**